### ISSUE TITLE
Please Share the Pre trained Models

### ISSUE THREAD

**Issue by yellowjs0304** [weituo12321/PREVALENT_R2R#3]
Hi, First of all thank you for sharing your nice work.

I'm really interested in your work.
In your readme(https://github.com/weituo12321/PREVALENT_R2R#train-agent-for-r2r), you shared the speaker and initial agent which is pre-trained in EnvDropout.

But to use the models, I found that it must have access for your drive... 
I already sent an e-mail about it, but I couldn't get any reply from you.
Could you please check it, and share the model please?
I downloaded the env-dropout and had a training time, so i got new pre-trained bt agent models.
But It is not suitable your code, it returns below errors.
![command](https://user-images.githubusercontent.com/30281608/88474873-c01d1780-cf65-11ea-8850-578df5f8d4d2.jpg)
I think it should be training in your code, but It takes too long to train the model.
Could you please share the models?

**Comment by yellowjs0304** [comment]
Sorry, I found that you already edited.

**Comment by tito21** [comment]
Hello @yellowjs0304
I am trying to finetune this model on a different dataset similarr to R2R. How did you mange to use the pretrained model?

**Comment by yellowjs0304** [comment]
@tito21 I'm sorry, it has been almost 2 years, I can't remember about this.
